src/gradescopeapi/_classes/_account.py

```diff
@@ -70,3 +77,124 @@ def get_courses(self) -> dict:
         courses["student"] = student_courses
 
         return courses
+
+    def get_assignments(self, course_id: int) -> List[Dict[str, str]]:
+        """
+        Get a list of detailed assignment information for a course
+        Returns:
+            list: A list of dictionaries, where the keys are the assignment info name and
+            the values is the corresponding assignment information
+            Example:
+                [
+                   {
+                        'assignment_id': 'a_id',
+                        'name': 'a_name,
+                        'release_date': 'a_release_date',
+                        'due_date': 'a_due_date',
+                        'late_due_date': 'a_late_due_date',
+                        'submissions_status': 'a_status',
+                        'grade': 'a_grade',
+                        'max_grade': 'a_max_grade'
+                   }
+                ]
```

Comment:
i think, to keep consistent with how Calvin did extensions and I did courses, the assignment attributes could be turned into a dataclass implemented in ```assignment.py```.  Inside the helper functions that scrape assignment info then, you could create these objects.